applications/vanilla/models/class.discussionmodel.php

```diff
@@ -710,6 +710,8 @@ public function getWhere($where = false, $orderFields = '', $orderDirection = ''
 
         $sql = $this->SQL;
 
+        $this->fireEvent('modifyQuery');
```

Comment:
If you need to introduce a new event, naming it "modifyQuery" is probably going to be too generic. There's isn't any context about what type of query you're attempting to modify (e.g. delete, update, insert). It would also be beneficial to know where it is being fired (e.g. some way to distinguish between the event being fired in `getWhere` versus `getCount`).